system/Language/en/Files.php

```diff
@@ -11,6 +11,8 @@
 
 // Files language settings
 return [
-    'fileNotFound' => 'File not found: {0}',
-    'cannotMove'   => 'Could not move file {0} to {1} ({2}).',
+    'fileNotFound'      => 'File not found: {0}',
+    'cannotMove'        => 'Could not move file {0} to {1} ({2}).',
+	'expectedDirectory' => '{0} expects a valid directory.',
+	'expectedFile'      => '{0} expects a valid file.',
```

Comment:
spacing need an enhancement?

Reply:
I'll run this whole thing (actually all of 4.2 branch) through CS Fixer after we release it. You can ignore code standard issues for now.